### PR TITLE
feat: added profile image with publickey that posted a message

### LINKED ISSUE #73: Display user name for each statement
This might possibly need some UX work, but the implementation is simple
image

Current view, a name being added using nostr metadata could be a good addition to see who made which post

- [x] Show names for each statement/ post
- [x] Show names at top level topic

### PR DESCRIPTION
**Pull request by cdxker**
image

Only displaying public key at the moment since displaying metadata will require TONS of connections.

### CODE REVIEW

File: src/components/Atoms/NostrUserName.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+            authors: [props.pubkey],
+            kinds: [0],
+          },
+        ],
+        {
+          skipVerification: true,
+        },
+      );
+
+      console.log("fetching user metadata");
```
Comment (kmkhami) [inline src/components/Atoms/NostrUserName.tsx:43]: Could remove the log

File: src/components/Atoms/NostrUserName.tsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+      metadataSub.on("event", (event: Event) => {
+        console.log("fetching user metadata");
```
Comment (skeptrunedev) [inline src/components/Atoms/NostrUserName.tsx]:
```suggestion
```

File: src/components/Atoms/NostrUserName.tsx (hunk 1 above)
Comment (skeptrunedev) [inline src/components/Atoms/NostrUserName.tsx]:
```suggestion
```

**Review by skeptrunedev — APPROVED** [review]
LGTM - will test on deployment

**Pull request merged**